### ISSUE TITLE
Check if the browser supports IntersectionObserver conditions are not enough

### ISSUE THREAD

**Issue by tong233** [wellyshen/react-cool-img#222]
Hi, on Chorme 57, window.IntersectionObserver exists but entry.isIntersecting is always undefined. Please add like below to prevent lazy loading. In order to use isIntersecting normally.
```
if (
  !window.IntersectionObserver ||
  (window.IntersectionObserverEntry && !('isIntersecting' in window.IntersectionObserverEntry.prototype)))
) {
  console.error(observerErr)
  return (): void => null
}
```

**Comment by wellyshen** [comment]
@tong233 Seems it's supported via Chrome 51 right?

<img width="928" alt="螢幕快照 2020-04-22 下午11 31 53" src="https://user-images.githubusercontent.com/21308003/80002274-02abff80-84f2-11ea-8b24-ccea5314cd47.png">

**Comment by wellyshen** [comment]
@tong233 This issue has been fixed in [v1.0.28](https://github.com/wellyshen/react-cool-img/releases/tag/v1.0.28), thank you.
